SPRCLI/Sources/SPR/UploadManifest.swift

```diff
@@ -38,7 +38,7 @@ extension SPRPublisher {
         let fileHandle = try FileHandle(forReadingFrom: manifestFileURL)
         let stream = FileStream(fileHandle: fileHandle)
         let body = ByteStream.stream(stream)
-        let input = PutObjectInput(body: body, bucket: bucket, contentType: "text/x-swift", key: manifestKey)
+        let input = PutObjectInput(body: body, bucket: bucket, cacheControl: "public, immutable", contentType: "text/x-swift", key: manifestKey)
```

Comment:
Manifests (like archives) should never, ever change once published so this cache-control is also set to immutable.